UKHomeOffice/ide-controller
# COP 3336: New score calculation

## Description
Correct percentage calculation to account for minimum score. 

## To test
- pull branch and run `npm ci && npm run build`
- deploy new build to IDE development machine
- run scan of document and allow idemia to return match score
- score will display in the top left corner, you should now be able to achieve a full range of % scores, with a passing score of 47%

## Developer Checklist

\* Required
